src/biz/workspace/publish.rs

```diff
@@ -113,6 +113,16 @@ pub async fn set_workspace_default_publish_view(
   Ok(())
 }
 
+pub async fn unset_workspace_default_publish_view(
+  pg_pool: &PgPool,
+  user_uuid: &Uuid,
+  workspace_id: &Uuid,
+) -> Result<(), AppError> {
+  check_workspace_owner(pg_pool, user_uuid, workspace_id).await?;
```

Comment:
If we use enforce role: Owner, we don't need to check workspace owner again.

Reply:
Removed. In addition, I have remove 'check_workspace_owner' and use `enforce_role` on the caller instead for other operation that requires `check_workspace_owner`